plugins.js

```diff
@@ -193,6 +193,9 @@ function install (fn) {
   shellEnv().then((env) => {
     let registry = exports.getDecoratedConfig().npmRegistry;
     if (registry) env.NPM_CONFIG_REGISTRY = registry;
+    env.npm_config_runtime = 'electron';
+    env.npm_config_target = '1.2.5';
```

Comment:
Maybe we can read the setting from our own build to use a matching electron version?